src/controllers/expenses.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+const { getUserById } = require('../services/users');
+const { isValidData } = require('../utils/expenses');
+
+const getAllExpenses = (req, res) => {
```

Comment:
This controller needs to parse query as [described here](https://app.swaggerhub.com/apis/YURIIHOLIUK_1/Accounting-App/1.0.0#/Expenses/getExpenses).
`category` can be an array - keep it in mind.
F.e.:
```http
GET /expenses?userId=1&category=Food&category=Sport&from=2022-10-01&to=2022-11-01
```
It means you need to filter all expenses for user1 in 2 categories between dates. All query params are optional, but you need to work with all.